ietf/api/views.py

```diff
@@ -449,18 +452,21 @@ def directauth(request):
 
 
         # Note well that we are using user.username, not what was passed to the API.
-        if user_query.count() == 1 and authenticate(username = user_query.first().username, password = password):
+        user_count = user_query.count()
+        if user_count == 1 and authenticate(username = user_query.first().username, password = password):
             user = user_query.get()
             if user_query.filter(person__isnull=True).count() == 1: # Can't inspect user.person direclty here
                 log.log(f"Direct auth of personless user {user.pk}:{user.username}")
             else:
                 log.log(f"Direct auth: {user.pk}:{user.person.plain_name()}")
+            log.log(f"Direct auth success: {username}")
```

Comment:
This is overly chatty - can we tighten it up so that we don't get two log lines as we go through this code?

Reply:
Good point. I added the `Direct auth success` prefix to each successful response above.